Code/Tools/ProjectManager/Source/Settings.cpp

```diff
@@ -61,6 +55,18 @@ namespace O3DE::ProjectManager
         return saved;
     }
 
+    bool Settings::OnSettingsChanged()
+    {
+        if (m_saveToDisk)
+        {
+            return Save();
+        }
+        else
+        {
+            return true;
+        }
+    }
```

Comment:
right now it doesn't appear anyone is using the return value of OnSettingsChange() you can probably just make it return void and simplify the contents